crates/solver/src/settlement_post_processing/mod.rs

```diff
@@ -1,5 +1,13 @@
-pub mod optimize_buffer_usage;
-pub mod optimize_unwrapping;
+use {
+    crate::solver::score_computation::ScoreCalculator,
+    anyhow::{Context, Result},
+    ethcontract::U256,
+    shared::external_prices::ExternalPrices,
+};
+
+mod optimize_buffer_usage;
+mod optimize_score;
+mod optimize_unwrapping;
 
 use {
```

Comment:
Somehow the imports got split. It should be modules first then all imports.

Reply:
Fixed.